pkg/kubelet/pleg/generic.go

```diff
@@ -275,6 +288,15 @@ func (g *GenericPLEG) relist() {
 				metrics.PLEGDiscardEvents.Inc()
 				klog.ErrorS(nil, "Event channel is full, discard this relist() cycle event")
 			}
+			// Log exit code of containers when they finished in a particular event
+			if events[i].Type == ContainerDied {
+				containerID, ok := events[i].Data.(string)
+				if ok {
+					if exitCode, ok := containerExitCode[containerID]; ok {
+						klog.V(2).Infof("GenericPLEG: container finished: container ID: %v, exit code: %v", containerID, exitCode)
```

Comment:
likely needs to be replaced with the structured logging

Reply:
yeah, since it is first time we emit log info in generic.go. For pleg event, the format is like "SyncLoop (PLEG):". Therefore, in here, it used "GenericPLEG:" as a start. 

Wondering do you have any structured logging format for reference ?

Reply:
`klog.V(2).Infof("GenericPLEG: container finished", "containerID", container ID,  "exitCode", exitCode)`

Reply:
Thanks, updated with suggested structured logging